tests/workbenches/conftest.py

```diff
@@ -0,0 +1,309 @@
+from __future__ import annotations
+
+import logging
+import time
+import traceback
+from typing import Callable, Generator
+
+import kubernetes.dynamic
+from kubernetes.dynamic import DynamicClient
+
+import ocp_resources.pod
+import ocp_resources.resource
+
+import pytest
+
+
+@pytest.fixture(scope="function")
+def function_resource_manager(admin_client: DynamicClient) -> Generator[KubeResourceManager, None, None]:
+    resource_manager = KubeResourceManager(admin_client)
+    yield resource_manager
+    resource_manager.destroy()
+
+
+class KubeResourceManager:
+    def __init__(self, privileged_client: DynamicClient):
+        self.privileged_client = privileged_client
+        self.resources: list[ocp_resources.resource.Resource] = []
+
+    def createResourceWithoutWait(self, client: DynamicClient, resource: ocp_resources.resource.Resource):
+        resource.client = self.privileged_client
+        resource.create()
+        self.resources.append(resource)
+
+    def createResourceWithWait(self, client: DynamicClient, resource: ocp_resources.resource.Resource):
+        resource.client = self.privileged_client
+        resource.create()
+        self.resources.append(resource)
+
+    def destroy(self):
+        for resource in self.resources:
+            # self.privileged_client.delete(resource)
+            resource.delete()
+
+
+class OdhAnnotationsLabels:
+    OPENSHIFT_DOMAIN = "openshift.io/"
+    ODH_DOMAIN = "opendatahub.io/"
```

Comment:
used in other places as well, please place under https://github.com/opendatahub-io/opendatahub-tests/blob/main/utilities/constants.py and re-use